src/lib/parse/contracts.js

```diff
@@ -243,12 +243,16 @@ type BodyMetadata = {
 const validateBody = (bodyDescriptor: BodyDescriptor, schema: JsonSchema, bodyType: 'request' | 'response', metadata: BodyMetadata): SchemaValidationResult => {
     let result: SchemaValidationResult;
     try {
-        // the validator throws an error with empty contract  
-        // this validates when you have an empty body in this case
-        if (!bodyDescriptor.body && !schema) {
-            result = { valid: true, niceErrors: [] };
+        // the validator throws an error with empty scenario body  
+        // this validates this case
+        if (!bodyDescriptor.body) {
+            if (schema) {
+                result = { valid: false, niceErrors: ["No response body found"] };
```

Comment:
This is just improving the error message to be more user friendly

Reply:
👍